utils.ts

```diff
@@ -0,0 +1,60 @@
+//
+// Calendar Utils
+//
+
+/**
+ * Calculates the number of weeks in the current month.
+ *
+ * @param {number} month The month.
+ * @param {number} year The year.
+ * @returns {number} The number of weeks in the month.
+ */
+export function weeksInMonth(month: number, year: number) {
+  // month = # (0 indexed), year = #
+  const firstDay = new Date(year, month, 1).getDay(); // day of the week of 1st
+  const daysInMonth = new Date(year, month + 1, 0).getDate();
+  return Math.ceil((firstDay + daysInMonth) / 7);
+}
+
+/**
+ * Generates the string representation of a date.
+ *
+ * @param {number} month The integer representation of the month.
+ * @param {number} day The integer representation of the day of the month.
+ * @param {number} year The integer representation of the year.
+ * @returns {string} The date as a string.
+ */
+export function getStringDate(
+  month: number,
+  day: number,
+  year: number,
+): string {
+  const mm = month.toString().padStart(2, "0");
+  const dd = day.toString().padStart(2, "0");
+  return year + "-" + mm + "-" + dd;
```

Comment:
For formatting strings, I'd probably lean to template strings nowadays. Concatenation does work  though! So no need to change -- https://www.w3schools.com/js/js_string_templates.asp

```suggestion
  return `${year}-${mm}-${dd}`;
```